web-app/src/services/API/internalAPI/apiResources/types.ts

```diff
@@ -1,6 +1,11 @@
+import { AxiosResponse } from 'axios';
+
 export interface AuthLoginResponse {
   data: Location;
 }
+export interface LogoResponse extends AxiosResponse {
```

Comment:
this extends look useless

Reply:
No it is not useless. LogoResponse is still AxiosResponse but with logo field in data.

Reply:
Look on `AuthLoginResponse` this don't extend `AxiosResponse` and working :P 
When You pass return pass to `.get<type>(...)` its automatically pass type to return data

`extends AxiosResponse` is useless